src/WasmDis.ts

```diff
@@ -448,6 +462,20 @@ export class WasmDisassembler {
     this._lines.push(this._buffer);
     this._buffer = '';
   }
+  private logStartOfFunctionBodyOffset() {
+    if (this.addOffsets) {
+      this._currentFunctionBodyOffset = {
+        start: this._currentPosition,
+      };
+    }
+  }
+  private logEndOfFunctionBodyOffset() {
+    if (this.addOffsets && this._currentFunctionBodyOffset?.start) {
```

Comment:
I don't think the `?.start` is necessary here and I think it's less clear since the test really hinges on whether _currentFunctionBodyOffset is null.

Reply:
thanks! I fixed this. Also, this syntax is not allowed in devtools code I think. npm run check throw for this.